dashboard/app/views/shared/_concept_trophy_block.html.haml

```diff
@@ -1,5 +1,6 @@
 %div{style: 'display: inline-block; width: 290px; position: relative; overflow: hidden;' + added_style}
   %h4 #{t('nav.popup.mastery')}
+  %a{name: 'trophies'}
```

Comment:
[anchor name attributes are deprecated](https://www.w3.org/TR/html5/obsolete.html#obsolete); 'trophies' should be an id